packages/main/src/Toast.js

```diff
@@ -0,0 +1,201 @@
+import Integer from "@ui5/webcomponents-base/dist/types/Integer.js";
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import { getRTL } from "@ui5/webcomponents-base/dist/config/RTL.js";
+import ToastTemplate from "./generated/templates/ToastTemplate.lit.js";
+import ToastPlacement from "./types/ToastPlacement.js";
+
+// Styles
+import ToastCss from "./generated/themes/Toast.css.js";
+
+// Static Constants
+const MAXIMUM_ALLOWED_TRANSITION_DURATION_IN_MILLISECONDS = 1000;
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-toast",
+	properties: /** @lends sap.ui.webcomponents.main.Toast.prototype */ {
+
+		/**
+		 * Defines the duration in milliseconds for which <code>ui5-toast</code>
+		 * remains on the screen before it's automatically closed.
+		 *
+		 * @type {Integer}
+		 * @defaultvalue 3000
+		 * @public
+		 */
+		duration: {
```

Comment:
Currently, you are free to set duration="-1", duration="0", the Toast blinks for a moment. I am not sure how to interpret such strange values. I would document that the minimum allowed value is, I don`t know, 100ms or event 500ms,  and even if the user sets smaller values we will set the transition-duration=100/500